lib/broccoli/glimmer-app.ts

```diff
@@ -50,16 +59,21 @@ const DEFAULT_TS_OPTIONS = {
 
 export interface OutputPaths {
   app: {
-    html: string
+    html: string,
+    js: string
   }
 }
-
 export interface EmberCLIDefaults {
   project: Project
 }
 
 export interface Options {
```

Comment:
Since this is a named export, perhaps this should be `GlimmerAppOptions` or just plain `AppOptions`? Obviously, there could be extensions of a base interface for both Ember and Glimmer apps.